app/controllers/rescue_requests_controller.rb

```diff
@@ -76,7 +76,7 @@ def apply_triage_status
   def apply_medical_triage_status
     prev_status = @request.medical_status&.name || "(no status)"
     if @request.update(medical_status_id: params[:status_id])
-      @request.case_notes.create(user_id: current_user.id, medical: true, content: "Changed status from #{prev_status} to #{@request.medical_status.name}")
+      @request.case_notes.create(user_id: nil, medical: true, content: "Changed status from #{prev_status} to #{@request.medical_status.name}")
```

Comment:
You probably want to mention the user who made the change in the content of the note - "User 'ArtOfCode' (ID 7) changed status..." or something

Reply:
I just added username, those should be unique enough.